style.css

```diff
@@ -569,29 +463,29 @@ nav > a {
   display: flex;
   flex-direction: column;
   padding: 24px;
-  background-color: #ffffff;
+  background-color: #fff;
   margin-top: 114px;
   border-top-right-radius: 80px;
 }
 
 .about-myself h1 {
   width: 327px;
   height: 35px;
-  font-family: 'Poppins';
+  font-family: Poppins, sans-serif;
   font-style: normal;
   font-weight: 700;
   font-size: 40px;
```

Comment:
- [ ] Kindly do the same for this too. Look at the screenshot and change the underline heading.

### Your About Myself

![your-about-h1](https://github.com/santitoot/finish-mobile-version/assets/26597432/b1cf8819-c494-4b87-8cd5-cbc117ce4143)

- [ ] It is also required that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. Kindly implement that too.

### Your Tech Categories
![your-about-languages](https://github.com/santitoot/finish-mobile-version/assets/26597432/12d89807-5b01-4462-b016-bd4c09da6212)


- [ ] You're required to use header tag (h2,......h6) for the subtitle `Let's Connect` in this section instead of the `p` tag. Kindly change that too.

### Your Let's Connect subtitle
![your-lets-connect](https://github.com/santitoot/finish-mobile-version/assets/26597432/e2a76e78-63aa-46ba-bd42-e4296fe07545)

Reply:
I made the required change